packages/babel-plugin-proposal-private-methods/README.md

```diff
@@ -0,0 +1,19 @@
+# @babel/plugin-proposal-private-methods
```

Comment:
I tried to follow what seemed like the norm for other packages here, let me know if there's some different process around this (i.e., creating a new plugin and the associated files).

Reply:
It's correct :+1: